handbook/support.md

```diff
@@ -0,0 +1,13 @@
+# Requesting Support
+
+## Community Forum
+If you have any question like how to do ____ i.e. how to setup SFTP in EasyEngine, you should post in [community forum](https://community.easyengine.io/). There are helpful folks around who might be able to guide you 🕵🏻‍♂️️.
+
+## Slack
+For sharing feedback on EasyEngine and for discussion related to EasyEngine development, you can come by and say hello to us on slack :wave: [![Join EasyEngine Slack Channel](http://slack.easyengine.io/badge.svg)](http://slack.easyengine.io/)
+
+## Github
+If you think you've found a bug in EasyEngine, you can create the issue in [one of our github repositories](github-repos.md).
```

Comment:
Again, go to the community forum. Once we verify a bug there, we will move it to Github issues. I want to keep issue tracker clean.